<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WiFiSettingWEPCell</class>
 <widget class="QStackedWidget" name="WiFiSettingWEPCell">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>220</width>
    <height>51</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StackedWidget</string>
  </property>
  <property name="currentIndex">
   <number>1</number>
  </property>
  <widget class="QWidget" name="page0">
   <widget class="QLabel" name="label_APStatus">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>21</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ResStr_Connected</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_line1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>42</y>
      <width>200</width>
      <height>1</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QLabel{
border:1px solid  rgb(181, 181, 181);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>1</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton{
	border:transparent;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_SSID1">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>0</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>SWQA</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_StatusIcon">
    <property name="geometry">
     <rect>
      <x>186</x>
      <y>25</y>
      <width>16</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	border-image: url(:/Images/Signal_Enable.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_APStatus</zorder>
   <zorder>label_line1</zorder>
   <zorder>label_StatusIcon</zorder>
   <zorder>label_SSID1</zorder>
   <zorder>pushButton</zorder>
  </widget>
  <widget class="QWidget" name="page1">
   <widget class="QRadioButton" name="btKey3">
    <property name="geometry">
     <rect>
      <x>103</x>
      <y>125</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>45</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton{	
	spacing: 1px;
}
QRadioButton::indicator {
    width: 20px;
    height: 20px;
}

QRadioButton::indicator::unchecked {
	image: url(:/Images/Uncheck.png);
}

QRadioButton::indicator::checked {
	image: url(:/Images/Checked.png);
}</string>
    </property>
    <property name="text">
     <string>key3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Password">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>48</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>ResStr_Password</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_keyid">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>104</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>WEP KeyID</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_APStatus_2">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>21</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ResStr_Connected</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btConnect">
    <property name="geometry">
     <rect>
      <x>102</x>
      <y>153</y>
      <width>90</width>
      <height>28</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>90</width>
      <height>28</height>
     </size>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-image:url(:/Images/Btn_Green.png);
	color: rgb(255, 255, 255);
}
QPushButton:hover{
	border-image:url(:/Images/Btn_Green.png);
	color:rgb(0,0,0);
}
QPushButton:pressed{
	border-image:url(:/Images/Btn_Green_Pressed.png);
	color: rgb(255, 255, 255);
}
</string>
    </property>
    <property name="text">
     <string>ResStr_Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>2</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton{
	border:transparent;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="btKey2">
    <property name="geometry">
     <rect>
      <x>53</x>
      <y>125</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>45</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton{	
	spacing: 1px;
}
QRadioButton::indicator {
    width: 20px;
    height: 20px;
}

QRadioButton::indicator::unchecked {
	image: url(:/Images/Uncheck.png);
}

QRadioButton::indicator::checked {
	image: url(:/Images/Checked.png);
}</string>
    </property>
    <property name="text">
     <string>key2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_SSID2">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>0</y>
      <width>191</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>SWQA</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="btKey4">
    <property name="geometry">
     <rect>
      <x>151</x>
      <y>125</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>45</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton{	
	spacing: 1px;
}
QRadioButton::indicator {
    width: 20px;
    height: 20px;
}

QRadioButton::indicator::unchecked {
	image: url(:/Images/Uncheck.png);
}

QRadioButton::indicator::checked {
	image: url(:/Images/Checked.png);
}</string>
    </property>
    <property name="text">
     <string>key4</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_visiable">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>73</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox{	
	spacing: 3px;
}
QCheckBox::indicator {
    width: 20px;
    height: 20px;
}

QCheckBox::indicator::unchecked {
	image: url(:/Images/Uncheck.png);
}

QCheckBox::indicator::checked {
	image: url(:/Images/Checked.png);
}</string>
    </property>
    <property name="text">
     <string>ResStr_Display_Password</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="btKey1">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>125</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>45</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton{	
	spacing: 1px;
}
QRadioButton::indicator {
    width: 20px;
    height: 20px;
}

QRadioButton::indicator::unchecked {
	image: url(:/Images/Uncheck.png);
}

QRadioButton::indicator::checked {
	image: url(:/Images/Checked.png);
}</string>
    </property>
    <property name="text">
     <string>key1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Password">
    <property name="geometry">
     <rect>
      <x>84</x>
      <y>49</y>
      <width>129</width>
      <height>20</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border:2px groove gray;
border-radius:5px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>26</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btCancel">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>153</y>
      <width>90</width>
      <height>28</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>90</width>
      <height>28</height>
     </size>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-image:url(:/Images/Btn_Gray.png);
}
QPushButton:hover{
	border-image:url(:/Images/Btn_Gray.png);
	color:green;
}
QPushButton:pressed{
	border-image: url(:/Images/Btn_Gray_Pressed.png);
	color:white;
}</string>
    </property>
    <property name="text">
     <string>ResStr_Cancel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_line2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>191</y>
      <width>200</width>
      <height>1</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QLabel{
border:1px solid  rgb(181, 181, 181);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>btKey3</zorder>
   <zorder>label_Password</zorder>
   <zorder>label_keyid</zorder>
   <zorder>label_APStatus_2</zorder>
   <zorder>btConnect</zorder>
   <zorder>btKey2</zorder>
   <zorder>label_SSID2</zorder>
   <zorder>btKey4</zorder>
   <zorder>checkBox_visiable</zorder>
   <zorder>btKey1</zorder>
   <zorder>lineEdit_Password</zorder>
   <zorder>btCancel</zorder>
   <zorder>label_line2</zorder>
   <zorder>pushButton_2</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
